From 4035a75e55848aabf66cde9b0b59b4adbc45db51 Mon Sep 17 00:00:00 2001
From: Guy Harris <guy@alum.mit.edu>
Date: Sun, 20 Dec 2015 18:21:35 -0800
Subject: [PATCH 90/90] Squelch another warning.

Change-Id: I7340954d9ca2fd11a6db2aa7cd5493d870181e23
Reviewed-on: https://code.wireshark.org/review/12765
Reviewed-by: Guy Harris <guy@alum.mit.edu>
(cherry picked from commit c9670e334c38f98da485b53bbd09571047836064)
Reviewed-on: https://code.wireshark.org/review/12767
(cherry picked from commit 8d9e4aad5ea5c6c946f213504c38ef01cc84d2e5)
Reviewed-on: https://code.wireshark.org/review/13777
Reviewed-by: Balint Reczey <balint@balintreczey.hu>
---
 wiretap/ngsniffer.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/wiretap/ngsniffer.c b/wiretap/ngsniffer.c
index 1201347..cb1b3d9 100644
--- a/wiretap/ngsniffer.c
+++ b/wiretap/ngsniffer.c
@@ -1264,7 +1264,7 @@ ngsniffer_process_record(wtap *wth, gboolean is_random, guint *padding,
 		 * record, so it's too short by (sizeof frame4 - sizeof frame2).
 		 */
 		if (ngsniffer->maj_vers < 5 && ngsniffer->min_vers >= 95)
-			rec_length_remaining += sizeof frame4 - sizeof frame2;
+			rec_length_remaining += (guint)(sizeof frame4 - sizeof frame2);
 
 		/* Do we have an f_frame4_struct worth of data? */
 		if (rec_length_remaining < sizeof frame4) {
-- 
2.1.4

